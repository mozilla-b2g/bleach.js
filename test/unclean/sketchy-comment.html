Foo bar.
<pre wrap=""><!---->
I should exist still!
</pre>
<div class="moz-signature">-- <br><br>
I am the footer.
</div>
