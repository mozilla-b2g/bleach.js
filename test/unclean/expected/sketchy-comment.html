Foo bar.
<pre>
I should exist still!
</pre>
<div class="moz-signature">-- <br/><br/>
I am the footer.
</div>
